anesthetic/samples.py

```diff
@@ -345,6 +347,64 @@ def plot_2d(self, axes, *args, **kwargs):
 
         return fig, axes
 
+    def credibility_interval(self, key, level=0.68, method="iso-probability"):
+        """Compute the marginalised credibility interval.
+
+        Also referred to as confidence interval or iso-probability contours.
+        Also options to compute upper or lower limits instead.
+
+        Note: This is based on the discrete set of points and weights without
+        interpolation. Discretization error is determined by the (normalized)
+        weights, i.e. is small for large sample sizes with small weights.
```

Comment:
Is this still true?

Reply:
I have now written up the details with a bunch of plots [here](https://github.com/Stefan-Heimersheim/fastCI), in particular here is the interpolated CDF that is used to determine the interval:
![CDF](https://github.com/Stefan-Heimersheim/fastCI/raw/main/illustrations/CDF_distances.png?raw=true)
What I meant with "without interpolation" was more like "without KDE", in fact I am using an interpolation of the CDF. I will adjust the description

Reply:
This is an excellent description. I've added a link to the docstring.